src/features/query-bar/components/filter/Condition.js

```diff
@@ -139,9 +172,7 @@ const Condition = ({
                 break;
         }
 
-        const error = areErrorsEnabled && !isValueSet ? message : null;
-
-        return error;
+        return message;
```

Comment:
it actually looks like you can have each switch case set the `messageText` and then return `<FormattedMessage {...messageText}/>` in line 175